src/test/java/integration/CanvasIntegrationImplIT.java

```diff
@@ -3,43 +3,139 @@
 import edu.byu.cs.canvas.CanvasException;
 import edu.byu.cs.canvas.CanvasIntegration;
 import edu.byu.cs.canvas.CanvasIntegrationImpl;
+import edu.byu.cs.canvas.model.CanvasAssignment;
+import edu.byu.cs.dataAccess.ConfigurationDao;
 import edu.byu.cs.dataAccess.DaoService;
-import edu.byu.cs.dataAccess.memory.QueueMemoryDao;
-import edu.byu.cs.dataAccess.memory.RubricConfigMemoryDao;
-import edu.byu.cs.dataAccess.memory.SubmissionMemoryDao;
-import edu.byu.cs.dataAccess.memory.UserMemoryDao;
+import edu.byu.cs.dataAccess.DataAccessException;
+import edu.byu.cs.dataAccess.RubricConfigDao;
+import edu.byu.cs.model.Phase;
+import edu.byu.cs.model.Rubric;
+import edu.byu.cs.model.RubricConfig;
 import edu.byu.cs.model.User;
 import edu.byu.cs.properties.ApplicationProperties;
+import edu.byu.cs.util.PhaseUtils;
 import org.junit.jupiter.api.*;
 
+import java.util.Collection;
+import java.util.Map;
 import java.util.Properties;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class CanvasIntegrationImplIT {
 
+    private CanvasIntegration canvasIntegration;
+    private CanvasIntegrationImpl.CourseInfoRetriever retriever;
+
     @BeforeAll
     public static void setUp() {
         loadApplicationProperties();
     }
 
     @BeforeEach
-    public void setUpEach() {
-        DaoService.setRubricConfigDao(new RubricConfigMemoryDao());
-        DaoService.setUserDao(new UserMemoryDao());
-        DaoService.setQueueDao(new QueueMemoryDao());
-        DaoService.setSubmissionDao(new SubmissionMemoryDao());
+    public void setUpEach() throws DataAccessException {
+        DaoService.initializeMemoryDAOs();
+        DaoService.getConfigurationDao().setConfiguration(
+                ConfigurationDao.Configuration.COURSE_NUMBER,
+                26822,
+                Integer.class
+        ); // FIXME: ??? Maybe get dynamically
```

Comment:
Not sure what to do here besides hardcoding the course number. Another potential thing is setting an environment variable like the canvas api token.

Reply:
I think an environment variable makes sense